gef-ui/package.json

```diff
@@ -22,14 +22,15 @@
     "api-test": "jest --coverage --verbose --forceExit --config=api-test.jest.config.js",
     "build": "rimraf ./public/js && rimraf ./public/css && webpack --mode production --config webpack.prod.config.js",
     "dev": "webpack --config webpack.dev.config.js --watch",
-    "lint": "eslint ./server ./scripts ./component-tests ./test-helpers ./api-tests",
+    "lint": "eslint ./server ./scripts ./component-tests ./test-helpers ./api-tests && npm run type-check",
```

Comment:
same thing here won't put it on all, I guess type-check only runs on typescript files so it's fine to be broad?